app/src/main/java/de/mytfg/app/android/gcm/GcmCallbackRegistration.java

```diff
@@ -29,5 +34,50 @@ public void callback(GcmNotification notification) {
                 }
             }
         });
+        // Set callback for Messages
+        MyTFG.gcmManager.setReceiveCallback("message", new GcmCallback() {
```

Comment:
Das hat ja nix mit dem Terminal zu tun und sollte daher in eine eigene Methode registerConversations()